connectors/katalog/example/asset.yaml

```diff
@@ -2,28 +2,24 @@ apiVersion: katalog.fybrik.io/v1alpha1
 kind: Asset
```

Comment:
BTW, there is another asset definition in https://github.com/fybrik/fybrik/blob/master/manager/testdata/notebook/example-asset.yaml which should be updated as well.